src/constants.ts

```diff
@@ -374,6 +374,20 @@ export enum TelemetryEventName {
     MICROBIT_PERFORMANCE_DEPLOY_DEVICE = "MICROBIT.PERFORMANCE.DEPLOY.DEVICE",
     MICROBIT_PERFORMANCE_NEW_FILE = "MICROBIT.PERFORMANCE.NEW.FILE",
     MICROBIT_PERFORMANCE_OPEN_SIMULATOR = "MICROBIT.PERFORMANCE.OPEN.SIMULATOR",
+
+    // Venv options
+    SETUP_VENV_CREATION_ERR = "SETUP.VENV.CREATION.ERR",
```

Comment:
Are you familiar with how this string is used in app insights?

Reply:
They're just strings. It's easier to search for related events this way and it was also the way that the previous interns did it.

Reply:
Thanks for clarifying!
I was thinking we could structure them in a more useful way (For e.g. everything related to setup starting with `SETUP.***`, all errors related to set up would be `SETUP.ERROR.***`, as opposed to having the name separated with dots instead of underscores `A_B_C_D` -> `A.B.C.D`) to streamline filtering events. Sayyeda showed me how it all looks in AppInsights and it seems there is no support for this kind of filtering out of the box (unless you write queries), so I think it's fine to keep it this way for now.

Reply:
Definitely a good idea